ci/test.msvc.sh

```diff
@@ -10,7 +10,8 @@ mkdir -p "build/Testing/Temporary"
 cp "cmake/CTestCostData.txt" "build/Testing/Temporary"
 
 cd build
-ctest -j"${CTEST_PARALLEL}" -C Debug --output-on-failure
+ctest -j"${CTEST_PARALLEL}" rnp_tests -C Debug --output-on-failure
```

Comment:
Looks like `ctest -j"${CTEST_PARALLEL}" -E "cli_tests" -C Debug --output-on-failure`
would do the trick. -E excludes those tests.

Reply:
@rrrooommmaaa It seems to be much simpler - I forgot to add `-R` parameter. Now it should go fine.